ci/run_tests.py

```diff
@@ -0,0 +1,75 @@
+import click
+import subprocess
+import os
+
+import pytest
+
+
+def run_pytest(lib, test_dir):
+    if RAPIDS_TESTS_DIR := os.environ.get("RAPIDS_TESTS_DIR", None):
+        junitxml = os.path.join(RAPIDS_TESTS_DIR, f"junit-{lib}.xml")
+    else:
+        junitxml = "/dev/null"
+
+    command = [
+        "pytest",
+        "-v",
+        "--continue-on-collection-errors",
+        "--cache-clear",
+        f"--junitxml={junitxml}",
+        *test_dir,
+    ]
+    try:
+        subprocess.run(command, check=True, capture_output=False)
+    except subprocess.CalledProcessError as e:
+        if e.returncode not in {pytest.ExitCode.OK, pytest.ExitCode.NO_TESTS_COLLECTED}:
+            raise e
+
+
+@click.command()
+@click.option("--ast_canopy", is_flag=True)
+@click.option("--numbast", is_flag=True)
+@click.option("--bf16", is_flag=True)
+@click.option("--fp16", is_flag=True)
+@click.option("--curand_device", is_flag=True)
+@click.option("--all_tests", is_flag=True)
+def run(
```

Comment:
Add documentation and help text for each of these options and the command line interface.